joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/utils/AstGenRunner.scala

```diff
@@ -173,9 +172,22 @@ object AstGenRunner {
 
 class AstGenRunner(config: Config) {
 
-  import io.joern.jssrc2cpg.utils.AstGenRunner._
+  import io.joern.jssrc2cpg.utils.AstGenRunner.*
 
-  private val executableArgs = if (!config.tsTypes) Seq("--no-tsTypes") else Seq.empty
+  private val executableArgs = {
+    val tsArgs = if (!config.tsTypes) Seq("--no-tsTypes") else Seq.empty
+    val ignoredFilesRegex = if (config.ignoredFilesRegex.toString().nonEmpty) {
+      Seq("--exclude-regex", config.ignoredFilesRegex.toString())
+    } else {
+      Seq.empty
+    }
+    val ignoreFileArgs = if (config.ignoredFiles.nonEmpty) {
+      Seq("--exclude-file") ++ config.ignoredFiles.map(f => s"\"$f\"")
```

Comment:
```suggestion
      Seq("--exclude-file") ++ config.ignoredFiles.map(_.toString)
```

Reply:
No, we need the extra quotes for paths with whitespaces.

Reply:
oh right, sorry I needed a coffe. done now :)